k8s/grafana/chart/grafana/templates/statefulset.yaml

```diff
@@ -70,6 +70,30 @@ spec:
             requests:
               cpu: 50m
               memory: 100Mi
+        {{ if .Values.metrics.enabled }}
+        - name: prometheus-to-sd
+          image: {{ .Values.metrics.image }}
+          ports:
+            - name: profiler
+              containerPort: 6060
+          command:
+            - /monitor
+            - --stackdriver-prefix=custom.googleapis.com
+            - --source={{ .Release.Name }}:http://localhost:3000/metrics
+            - --pod-id=$(POD_NAME)
+            - --namespace-id=$(POD_NAMESPACE)
+            - --scrape-interval=5s
```

Comment:
Please replace `--scrape-interval` and `--export-interval` args with `--monitored-resource-types=k8s`.

For example: https://github.com/GoogleCloudPlatform/click-to-deploy/pull/529/files#diff-fff20dd2de7dad7815fedb0981c7c594R127